include/AutomationPattern.h

```diff
@@ -76,14 +76,17 @@ class EXPORT AutomationPattern : public TrackContentObject
 	MidiTime timeMapLength() const;
 	void updateLength();
 
-	MidiTime putValue( const MidiTime & _time, const float _value,
-						const bool _quant_pos = true );
+	MidiTime putValue( const MidiTime & time,
+				const float value,
+				const bool quantPos = true,
+				const bool controlKey = false );
```

Comment:
You have two choices:
1. The clean way: Move the feature to the view object, which reacts to the control key.
2. The easy way: Keep the feature in the model and rename `controlKey` to something like `ignoreSurroundingPoints`.